class.jetpack.php

```diff
@@ -3162,6 +3162,11 @@ public static function plugin_activation( $network_wide ) {
 
 		update_option( 'jetpack_activation_source', self::get_activation_source( wp_get_referer() ) );
 
+		// Define Status as "Unknown"
```

Comment:
"Initializing"  sorry last commit changed the code on 3167 but not here in the comment